bolt/lib/Profile/DataAggregator.cpp

```diff
@@ -175,11 +180,16 @@ void DataAggregator::start() {
 
   findPerfExecutable();
 
-  if (opts::BasicAggregation) {
-    launchPerfProcess("events without LBR",
-                      MainEventsPPI,
+  if (opts::ArmSPE) {
+    // pid    from_ip      to_ip        predicted/missed not-taken?
```

Comment:
Can simplify to:

```suggestion
    // pid    from_ip      to_ip      flags
```

and then expand 'flags' below:
- P/N: whether branch was Predicted or Mispredicted
- N: optionally appears when the branch was Not-Taken (ie fall-through)

Reply:
Added the suggested details.